src/Discord.Net.Core/Entities/Messages/IUserMessage.cs

```diff
@@ -57,6 +57,18 @@ public interface IUserMessage : IMessage
         /// </returns>
         Task UnpinAsync(RequestOptions options = null);
 
+        /// <summary>
+        ///     Publish (crosspost) this message.
```

Comment:
Use third-person present tense for summary (i.e., `Publishes`/`crossposts`)

Reply:
Corrected to standard tense.